airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -305,8 +305,8 @@ void testCheckSuccess() throws Exception {
   @Test
   void testCheckFailure() throws Exception {
     ((ObjectNode) config).put(JdbcUtils.PASSWORD_KEY, "fake");
-    final AirbyteConnectionStatus actual = source.check(config);
```

Comment:
same as above. Unless I am missing something, this is not a functional change and is not needed.

Reply:
You're right, this is mainly a byproduct of mass "find and replace" functionality of using `AirbyteConnectionStatus status` to have the naming similar to other usages of `AirbyteConnectionStatus` variables. I can revert if desired, I mainly made changes to all usages of `AirbyteConnectionStatus actual` to `AirbyteConnectionStatus status`

There were some tests where the usage of `actual` made sense to me where it had `assertEqual(actual, expected)`